### PR TITLE
Improve the design of the "Guides" page

### PR DESCRIPTION
**Pull request by alif-munim**
Signed-off-by: Alif Munim <alif.munim@ryerson.ca>

## What type of PR is this ? 

- [ ] Bug fix
- [x] Enhancement

## What does this PR do ?
* Make guide titles blue to make their interactivity more apparent
* Add objective and image metadata to the guide markdown files
* Access and display objective and image metadata on each guide preview
* Remove "read more" link to eliminate redundancy
* Adjust spacing

## Which issue(s) does this PR fix ?
https://github.com/eclipse/codewind/issues/2929
_(Individual guide pages referenced in the issue not yet fixed.)_

## Does this PR require a documentation change ?
Not as far as I am aware.

## Any special notes for your reviewer ?
None.

### CODE REVIEW

File: docs/css/guides.css [hunk 1]
```diff
@@ -32,6 +33,9 @@
 
 .guides-post-row {
     width: 100%;
+    background-color: #f7f7f7;
+    padding: 20px;
```
Comment (jcockbain) [inline docs/css/guides.css:37]: My preference would be more padding/margin round the edge, to make it look more consistent with blogs.

File: docs/_layouts/guides.html [hunk 2]
```diff
@@ -32,6 +32,31 @@ <h2 class="guides-post-title"><a href="{{ post.url | relative_url }}"
                                 <a href="{{ post.url | relative_url }}" class="read-more-link">Read more</a>
                             </div>
                         </div>
+                    </div> -->
+
+                    <div class="media">
```
Comment (jcockbain) [inline docs/_layouts/guides.html:37]: I like the layout of the cards, and think the objectives look great now. 

One personal bit of feedback, is on mobile I'd prefer the text to be left/centre aligned to take up more of the card.

Screenshot 2020-06-03 at 14 34 27

File: docs/_guides/codewind-vs-code-quick-guide.md [hunk 3]
```diff
@@ -3,10 +3,12 @@ layout: guide
 summary_title: "Codewind in VS Code"
 title: "Getting Started with Codewind in VS Code"
 categories: guides
-guide_description: "Take advantage of Codewind's tools to help build high quality cloud native applications regardless of which IDE or language you use."
+description: "Take advantage of Codewind's tools to help build high quality cloud native applications regardless of which IDE or language you use."
 permalink: codewind-vscode-quick-guide.html
 duration: 5 minutes
 keywords: Codewind, VS Code, microservice
+image: "https://upload.wikimedia.org/wikipedia/commons/thumb/9/9a/Visual_Studio_Code_1.35_icon.svg/1200px-Visual_Studio_Code_1.35_icon.svg.png"
```
Comment (jcockbain) [inline docs/_guides/codewind-vs-code-quick-guide.md:10]: There's icons as part of https://github.com/eclipse/codewind-docs/pull/611 for both vs-code and eclipse.

**Comment by alif-munim** [comment]
Forgot to add a sign-off statement on one of my commits; attempting to close this PR and create a new one.

**Comment by jcockbain** [comment]
`git commit -s --amend` will amend the commit locally with a sign off... You'll then need to `git push --force` to push the amended commit as the history will have changed. That would have meant you didn't have to close and open a new PR.

**Pull request closed without merging**